chrysom/client.go

```diff
@@ -106,6 +107,10 @@ type ClientConfig struct {
 	// (Optional) If section is not provided with Listener, this
 	// feature will be disabled for the client.
 	Listen ListenerConfig
+
+	// HeaderConfig provides the placeholder names to be used in logger for traceID
+	// and spanID. (Optional) If HeaderConfig is not provided the defaults will be used.
+	HeaderConfig candlelight.HeaderConfig
```

Comment:
No need to address it here but I left a question on candlelight related to this: https://github.com/xmidt-org/candlelight/issues/18